runtime/compiler/runtime/GPUHelpers.cpp

```diff
@@ -1097,7 +1097,11 @@ static GpuMetaData* getGPUMetaData(uint8_t* startPC)
    {
       J9VMThread *vmThread = jitConfig->javaVM->internalVMFunctions->currentVMThread(jitConfig->javaVM);
       TR_MethodMetaData * metaData = jitConfig->jitGetExceptionTableFromPC(vmThread, (UDATA)startPC);
+#if defined(J9VM_OPT_SNAPSHOTS)
+      return J9JIT_DATACACHE_OFFSET_TO_PTR(metaData->gpuCodeOffset, GpuMetaData);
+#else /* defined(J9VM_OPT_SNAPSHOT) */
       return (GpuMetaData*)metaData->gpuCode;
+#endif
```

Comment:
```suggestion
      return (GpuMetaData*)J9JITEXCEPTIONTABLE_GPUCODE_GET(metaData);
```

Reply:
This is more cleaner, thanks a lot, made changes in https://github.com/eclipse/openj9/compare/9c44eed9f5b8ca00f0928f924fa08d1eca801a75..fd6b80febcdf82c159e03c19bf3a27ce611ee83b